### ISSUE TITLE
Note edits not saved and notes completely wiped

### ISSUE THREAD

**Issue by SnorrenEnzo** [CarnetApp/CarnetNextcloud#139]
Please report every bug related to the Android application here
https://github.com/PhieF/CarnetAndroid/

**Describe the bug**
When using the Carnet application in the Nextcloud environment in a browser, notes are often not correctly saved or even wiped. For example, I had one note with a todo list and some extra text. When editing the text and then closing the note, this new text is not saved to the note. A few tries later and the entire note was wiped, being completely empty. I did try to have the note completely loaded and then saved automatically, by keeping the small loading bar at the top in mind. However this did not help. It most often happens in the browser, but sometimes also in the Android app. It makes taking notes a very risky business because you can easily loose large notes, possibly loosing weeks of work.

Nextcloud version 18.0.4

**To Reproduce**
1. Create or edit a note
2. Close the note
3. Reopen the note

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - Browser: firefox
 - Version 0.23.7

**Smartphone (please complete the following information):**
 - Device: Xiaomi Mi 5s
 - OS: LineageOS 16.1
 - Version 0.23.1

**Comment by PhieF** [comment]
Hi, are you using nginx ? I had this problem recently when switching to nginx

thanks for your report

**Comment by SnorrenEnzo** [comment]
Yes I am using Nginx! Forgot to mention that it is running in a FreeNAS jail.

**Comment by PhieF** [comment]
OK, to me it was a caching issue in the nginx configuration, I get it working by updating with the latest version of nginx conf provided by nextcloud 

Could you please check the differences you have ?

https://docs.nextcloud.com/server/latest/admin_manual/installation/nginx.html

**Comment by SnorrenEnzo** [comment]
I updated Nextcloud to 18.0.12 and changed the nginx conf to the one you linked to, with the server name, location of keys and root adapted to my setup. I still have the same issues. Will try a Nextcloud update to 19.0.6 next.

**Comment by SnorrenEnzo** [comment]
The issue still persists, I just had one of my lengthy notes wiped. Is there any other option I could try?

**Comment by PhieF** [comment]
could you check your network tab

for example on firefox, press f12
go to network tab, 
open a new note
write some text, and send me lines containing "saveText"

![Capture d’écran de 2021-01-14 19-11-41](https://user-images.githubusercontent.com/26841885/104631375-7273a300-569c-11eb-8f3c-4ad257ae8de7.png)

**Comment by SnorrenEnzo** [comment]
Sorry for leaving you having for so long! I thought I could work around the issues, but they have been popping up again. Here is an anonymized HAR file. I cut a lot of sensitive information out, as well as all the note details around `"text": "{\"metadata\":{"` so the file contains only roughly the first section of the original file. The notes itself are nearly 400 kB uncompressed. The original file is ~670 kB.

[network.har.txt](https://github.com/CarnetApp/CarnetNextcloud/files/6784420/network.har.txt)
